src/utils/fxa.js

```diff
@@ -129,7 +129,7 @@ async function deleteSubscription(bearerToken) {
         }
       })
       if (!response.ok) {
-        // throw new InternalServerError(`bad response: ${response.status}`)
+        // throw new InternalServerErrorg(`bad response: ${response.status}`)
```

Comment:
Is this meant to be commented out? Seems to have a typo too ("Errorg")

Reply:
Not sure why it's commented out, but I adjusted it so it throws and should be caught by the `catch` below